Makefile

```diff
@@ -118,6 +118,16 @@ KPROMO_PKG := sigs.k8s.io/promo-tools/v3/cmd/kpromo
 CONVERSION_VERIFIER_BIN := conversion-verifier
 CONVERSION_VERIFIER := $(abspath $(TOOLS_BIN_DIR)/$(CONVERSION_VERIFIER_BIN))
 
+OPENAPI_GEN_VER := v0.0.0-20211115234752-e816edb12b65
+OPENAPI_GEN_BIN := openapi-gen
+# We are intentionally using the binary without version suffix, to avoid the version
+# in generated files.
```

Comment:
I assume that's not just copy&paste and we had the same issue here as we had with conversion-gen?

Reply:
it is c&p, removed the comment and adjusted to use binary named by version.

Reply:
I was wrong, we have to keep as is, otherwise the name would be reflected in the generated files 👍